Assets/MixedRealityToolkit.SDK/Features/Audio/Influencers/AudioInfluencerController.cs

```diff
@@ -151,7 +151,9 @@ public float NativeHighPassCutoffFrequency
         }
 
         private void Awake()
-        {           
+        {
+            audioSource = GetComponent<AudioSource>();
```

Comment:
For many of these, the field will likely have

```
[SerializeField]
[HideInInspector]
```

which could be removed as well if we're moving to this pattern.